cli/cli.go

```diff
@@ -310,6 +347,10 @@ func setUsageExamples(info *usage.Info) {
 		"info powerbox",
 		"Show detailed info about the repository",
 	)
+	info.AddExample(
+		"render /etc/vgrepo/templates/default.tpl index.html",
```

Comment:
Perhaps better change command syntax to `render {output} {template}` and use default template for rendering?